5-testing-components/exercises-write-components/game-board/game-board.js

```diff
@@ -4,4 +4,20 @@
  * @param {Array[]} arrayOfArrays - a 2D array representing the game board
  * @returns {HTMLTableElement} the rendered game board
  */
-export const gameBoard = (arrayOfArrays) => {};
+export const gameBoard = (arrayOfArrays) => {
+const table = document.createElement('table');
+
+for (const array of arrayOfArrays) {
+    const tr = document.createElement('tr');
+    table.appendChild(tr);
+    for (const element of array) {
+        const td = document.createElement('td');
+        td.innerText = element;
+        tr.appendChild(td);
+    }
+table.appendChild(tr);
+  }
```

Comment:
When we have an arrayOfArrays represeting a table, usually we use names like `row` and `column` in the `for` loops.

```suggestion
for (const row of arrayOfArrays) {
    const tr = document.createElement('tr');
    table.appendChild(tr);
    for (const column of row) {
        const td = document.createElement('td');
        td.innerText = column;
        tr.appendChild(td);
    }
  }
```